src/forms/RegisterForm.js

```diff
@@ -0,0 +1,96 @@
+import './AddGameForm.scss'
+import './Form.scss'
+import React, {useState} from "react";
+
+function RegisterForm() {
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [repeatPassword, setRepeatPassword] = useState('');
+    const [passwordsMatch, setPasswordsMatch] = useState(true);
+
+    const handlePasswordChange = (e) => {
+        setPassword(e.target.value);
+    
+        // Check if passwords match and update passwordsMatch state
+        setPasswordsMatch(e.target.value === repeatPassword);
+      };
+    
+      const handleRepeatPasswordChange = (e) => {
+        setRepeatPassword(e.target.value);
+    
+        // Check if passwords match and update passwordsMatch state
+        setPasswordsMatch(e.target.value === password);
+      };
+
+    return (
+        <div className="add-game-form">
```

Comment:
nazwa tej klasy nie ma sensu, jeśli reużywamy jakieś klasy z innych modułów, to powinniśmy uogólnić ich nazwę